source/DomainTheory/BasesAndContinuity/ScottDomain.lagda

```diff
@@ -0,0 +1,114 @@
+---
+title:       Bounded completeness of a dcpo
+author:      Ayberk Tosun
+start-date:  2023-10-26
+---
+
+Ayberk Tosun.
```

Comment:
Do you need lines 7-9 if you have lines 1-5? By the way, if we adopt your header-suggestion, I would suggest to add a new field `dates-modified:` as well. In any case, we need to discuss this better on another occasion.

Reply:
> Do you need lines 7-9 if you have lines 1-5?

I also included lines 7-9 because the preamble in lines 1-5 is not a convention that we agreed on.

The idea was that, even if we end up not adopting this kind of a convention, it doesn’t hurt to put in a bit of additional information that can be ignored by the reader. However, I can remove it if you don’t like it.

I plan to implement a prototype of formatted HTML generation on my own development, and I need this information to be available in a standard format so that I can e.g. display the title as the title of the HTML page.